haveaqiupill/volunteer-app
# Add user data to user context

- Deployed latest master branch
- Add prettier config based on my current personal config, feel free to change it
- Add volunteer/researcher data to user context
- Add sign out button
- Support getting all programs by a researcher
-------
```
const user = useContext(UserContext);
```

- `user === null` means not logged in
-  `user?.uid` for user ID
- `user?.email` for user email
- `user?.data` for user's data (those in the user's document in firebase)
- `user?.isResearcher()` to check if user is a researcher
- `user?.isVolunteer()` to check if user is a volunteer
- Will add more functions as we go along
--------
- Will add applying research program stuff tmr
